src/pages/en/guides/rss.mdx

```diff
@@ -141,35 +141,41 @@ items: import.meta.glob('./blog/*.{md,mdx}'),
 
 The `content` key contains the full content of the post as HTML. This allows you to make your entire post content available to RSS feed readers.
 
+:::caution
+These examples focus on rendering Markdown contents. If you are using MDX, you will need to compile MDX contents to HTML manually. We recommend standard Markdown parsers like [Markdown-it](https://github.com/markdown-it/markdown-it) to get started, though this will not resolve JSX expressions or components.
+:::
+
+When using content collections, you can render post contents by calling the post's `render()` function and using the `html` property. Note the `html` property **is not present** for MDX entries. If you need to render MDX content, you can compile your MDX manually using the post's `body` property.
```

Comment:
Using content collections isn't *required* for full-post content right? We had this before? (Since 1.6.14)

This seems like when this page was updated, it was updated to now *only* show an example with content collections. I think it's important to show both, especially since this our blog tutorial includes creating an RSS feed but not using collections. So while we're here and updating the example, let's find a way to re-incorporate an example that works with globbing files in `src/pages/`, assuming that still works.

This was the original example for full post content. Will this (or an updated version of this) still work for projects not using collections?

```js ins={2, 16} title={src/pages/rss.xml.js}
import rss from '@astrojs/rss';
import sanitizeHtml from 'sanitize-html';
// Works with Markdown files only!
const postImportResult = import.meta.glob('../posts/**/*.md', { eager: true }); 
const posts = Object.values(postImportResult);
export const get = () => rss({
  title: 'Buzz’s Blog',
  description: 'A humble Astronaut’s guide to the stars',
  site: import.meta.env.SITE,
  items: posts.map((post) => ({
    link: post.url,
    title: post.frontmatter.title,
    pubDate: post.frontmatter.pubDate,
    content: sanitizeHtml(post.compiledContent()),
  }))
});
```